### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ezewer**
### Description:
Add a new endpoint, logins, that receives start, end, limit params and returns the registered logins for the user.

### New features:
- Add new endpoint, logins

### PR status:
- Version bumped
- Change-log updated
- Tests added
- Documentation updated

### Dependecies
- https://github.com/bitfinexcom/bfx-api-node-models/pull/32
- https://github.com/bitfinexcom/bfx-api-mock-srv/pull/27

### CODE REVIEW

File: lib/rest2.js [hunk 1]
```diff
@@ -619,6 +620,22 @@ class RESTv2 {
...
 
+  /**
+   * @param {number} start
+   * @param {number} end
+   * @param {number} limit
+   * @param {Method} cb
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-logins-hist
+   */
+  logins (start = 0, end = Date.now(), limit = 25, cb) {
```
Comment (JacobPlaster) [inline lib/rest2.js]: Could you copy the same style as `candles` and use an object as the argument please?

**Comment by JacobPlaster** [comment]
Please re-push to trigger travis. It should pass now that https://github.com/bitfinexcom/bfx-api-node-models/pull/32 has been published to npm

**Comment by JacobPlaster** [comment]
Travis is still having a hard time with this branch

image

**Comment by ezewer** [comment]
Some one with the right permissions might be able to reload Travis without sending a new commit https://stackoverflow.com/questions/17606874/trigger-a-travis-ci-rebuild-without-pushing-a-commit 
Might be required to reload all dependencies of travis also https://docs.travis-ci.com/user/caching/

**Pull request merged**